docs/UserGuide.md

```diff
@@ -221,7 +221,7 @@ Adds a new event, which can be optionally linked to any number of existing frien
 ### Editing an event: `editevent` / `ee`
 Edits an existing event.
 
-**Format**: `editevent INDEX [n/EVENT_NAME] [dt/DATE_TIME] [d/DESCRIPTION] [af/ADD_FRIEND_NAME]... rf/[REMOVE_FRIEND_NAME]...`
+**Format**: `editevent INDEX [n/EVENT_NAME] [dt/DATE_TIME] [d/DESCRIPTION] [af/ADD_FRIEND_NAME]... [rf/REMOVE_FRIEND_NAME]...`
```

Comment:
Think this format is still incorrect, because `INDEX` and `EVENT_NAME` are exclusively or. Should be `INDEX ? n/EVENT_NAME`

Reply:
My mistake!